style.css

```diff
@@ -98,3 +103,241 @@ a {
 p a {
   color: #7f8cff;
 }
+
+/* signig worection */
+.workcards {
+  display: grid;
+  row-gap: 2em;
+  padding: 100px 25px;
+}
+
+.card {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 20px;
+  min-height: 80vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.pic1 img {
+  width: 100%;
+}
+
+.title1 {
+  font-family: Poppins, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+}
+
+.frame-container {
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  margin-top: -12px;
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  padding: 0%;
+  margin-left: -5px;
+}
+
+.role,
+.year {
+  color: #7a869a;
+}
+
+.card-text {
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.language-container {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin-top: 0;
+}
+
+.language {
+  font-family: Poppins, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.html,
+.css,
+.java {
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 12px;
+  padding: 10px;
+}
```

Comment:
- [ ] Good job so far 👍👍 I noticed that your design has a bit more `padding` and a bit more `border-radius`. In the Figma template, it is `padding: 4px 12px;` and `border-radius: 8px;` Kindly fix this as well because of the following requirement ✨🤓
- [ ] Also, kindly consider aligning the list item with the paragraph as given in the Figma template. This can be fixed by removing `margin-left` from `.html` only

> _You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)._

Your design | Given design
---| ---|
![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/2a6ea668-0a86-428a-b54c-e3c4e052278b) | ![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/1736148b-b1d7-4785-8194-de1283151cdd) 

_Kindly apply this to all project cards_